book/src/06_system_data.md

```diff
@@ -37,17 +37,17 @@ It implements `SystemData` so just put it in your `SystemData` tuple.
 Please note that you may never write to these `Entities`, so only
 use `Read`. Even though it's immutable, you can atomically create
 and delete entities with it. Just use the `.create()` and `.delete()`
-methods, respectively. After dynamic entity creation / deletion,
+methods, respectively. After dynamic entity deletion,
 a call `World::maintain` is necessary in order to make the changes
```

Comment:
"call to" would be more appropriate here.  I know you didn't change this but I'd appreciate it if you fixed it :P